src/views/udns/form/components/UDNLayer2Parameters.tsx

```diff
@@ -39,14 +63,51 @@ const Layer2Parameters: FC = () => {
             />
           )}
         />
+        <FormGroupHelperText>
+          {t('IPAMLifecycle controls IP addresses management lifecycle.')}
+        </FormGroupHelperText>
       </FormGroup>
-      <FormGroup label={t('MTU')}>
+      <FormGroup
+        label={
+          <Popover
+            bodyContent={t(
+              'MTU is optional, if not provided, the globally configured value in OVN-Kubernetes (defaults to 1400) is used for the network.',
+            )}
+          >
+            <label className="pf-v5-c-form__label">{t('MTU')}</label>
+          </Popover>
+        }
+      >
         <TextInput
           type="number"
           {...register(`${TopologyKeys.Layer2}.mtu`, { max: 65536, min: 0, required: false })}
         />
+        <FormGroupHelperText>
+          {t('MTU is the maximum transmission unit for a network.')}
+        </FormGroupHelperText>
       </FormGroup>
-      <FormGroup label={t('Role')}>
+      <FormGroup
+        label={
+          <Popover
+            bodyContent={
+              <Flex direction={{ default: 'column' }}>
+                <FlexItem>
+                  {` - ${t(
+                    'Primary network is automatically assigned to every pod created in the same namespace.',
+                  )}`}
+                </FlexItem>
+                <FlexItem>
+                  {` - ${t(
+                    'Secondary network is only assigned to pods that use k8s.v1.cni.cncf.io/networks annotation to select given network.',
+                  )}`}
+                </FlexItem>
+              </Flex>
```

Comment:
And here also [list](https://v5-archive.patternfly.org/components/list/html/#ws-page-title)